src/main/java/com/scopely/mapper/JsonDynamoMapper.java

```diff
@@ -66,15 +70,19 @@
  */
 @SuppressWarnings({"WeakerAccess", "unused"})
 public class JsonDynamoMapper {
+    private final int BATCH_SAVE_UNPROCESSED_MAX_TRIES = 20;
```

Comment:
20 seems pretty high for retries. Maybe we could do something like `3`?

Reply:
Sure, changed it to 3.